Examples/options/overlaySignalAndPileup.py

```diff
@@ -0,0 +1,68 @@
+
+from Gaudi.Configuration import *
+
+
```

Comment:
Can you add few more comments to that file? What if signalCollections, what happens to them, what is merged etc? This is one of the examples so it may be good to add more documentation.